Slack.NetStandard/WebApi/TeamApi.cs

```diff
@@ -14,45 +14,14 @@ public TeamApi(IWebApiClient client)
             Billing = new TeamBillingApi(client);
         }
 
-       public Task<TeamAccessLogResponse> AccessLogs(long before)
-       {
-           return AccessLogs(before, null, null);
-       }
-
-        public Task<TeamAccessLogResponse> AccessLogs(int count, int page)
+        public Task<TeamAccessLogResponse> AccessLogs(TeamAccessLogRequest request)
         {
-            return AccessLogs(null, count, page);
+            return _client.MakeUrlEncodedCall<TeamAccessLogResponse>("team.accessLogs", request);
         }
 
-        public Task<TeamAccessLogResponse> AccessLogs(long? before, int? count, int? page)
```

Comment:
The `list` methods have always been annoying as there are standard patterns we use for those - so don't mind having a request objects as replacements.

In this instance there's no advantage to removing the simple methods, so let's keep those working please (don't mind if they use your request object method to be implemented, but let's keep them in the interface)

Reply:
Done. I restored the previous contract with the optional team_id parameter for all of them.